kirkstone.xml

```diff
@@ -40,6 +42,9 @@
   </project>
   <project revision="1c107bc6dc6437be87fbcd6a91528dba316384cf" remote="freescale" upstream="kirkstone"   name="meta-freescale-ml"       path="sources/meta-freescale-ml"/>
   <project revision="9e94b64bdfebcf7bfdf2af6447cec866a4efa814" remote="freescale" upstream="kirkstone"   name="meta-freescale-3rdparty" path="sources/meta-freescale-3rdparty"/>
-</manifest>
-
+  <project revision="260e3adc2bf322f52d81c0642c825088a88bb051" remote="yocto"  upstream="kirkstone" name="meta-arm" path="sources/meta-arm"/>
+  <project revision="c57b63147d2ac0dde16e43f7407d0ff50e62efdb" remote="yocto"  upstream="kirkstone" name="meta-ti" path="sources/meta-ti"/>
 
+  <project revision="0cb7188573584dd9db39e5689e843edbe9440588" remote="tdx" upstream="kirkstone-6.x.y" name="meta-toradex-bsp-common.git" path="sources/meta-toradex-bsp-common"/>
+  <project  revision="kirkstone-6.x.y"  remote="tdx" upstream="kirkstone-6.x.y" name="meta-toradex-ti.git" path="sources/meta-toradex-ti"/>
```

Comment:
```suggestion
  <project revision="kirkstone-6.x.y" remote="tdx" upstream="kirkstone-6.x.y" name="meta-toradex-ti.git" path="sources/meta-toradex-ti"/>
```

Reply:
Hmm, shouldn't it be kirkstone-6.x.y (floating) only on kirkstone-next and a specified version on kirkstone?

Reply:
correct, now fixed